services/catalog/src/simcore_service_catalog/core/events.py

```diff
@@ -23,11 +23,16 @@ async def start_app() -> None:
 
         # setup connection to pg db
         if app.state.settings.postgres.enabled:
+            # FIXME: check postgres service is in place and ready. Hand-shake?
             await connect_to_db(app)
 
         # setup connection to director
         if app.state.settings.director.enabled:
             setup_director(app)
+            # FIXME: check director service is in place and ready. Hand-shake??
+            # TEMP: added a dependency and waits 3 secs
+            import asyncio
+            await asyncio.sleep(3)
```

Comment:
please also add a log message saying something on the lines of "please remember to remove me when you have a better way of doing this"